gdp/column/gdp_column.py

```diff
@@ -554,64 +549,71 @@ def feed_vap_enthalpy_expr(_, c):
             k['B'] * (m.T_feed ** 2 - m.T_ref ** 2) / 2 +
             k['C'] * (m.T_feed ** 3 - m.T_ref ** 3) / 3 +
             k['D'] * (m.T_feed ** 4 - m.T_ref ** 4) / 4 +
-            k['E'] * (m.T_feed ** 5 - m.T_ref ** 5) / 5) * 1E-3
+            k['E'] * (m.T_feed ** 5 - m.T_ref ** 5) / 5) * 1E-3  # Feed vapor enthalpy = f(T_feed)
```

Comment:
But this is not true, in the following line you define the constraints to be equal to `m.H_V_spec_feed[c]`